articles/android-recyclerviews/index.md

```diff
@@ -0,0 +1,235 @@
+---
+layout: engineering-education
+status: publish
+published: true
+url: /engineering-education/android-recyclerviews/
+title: Android Recyclerviews
+description: This article goes through
+author: linus-muema
+date: 2020-11-04T00:00:00-10:00
+topics: [Languages]
+excerpt_separator: <!--more-->
+images:
+
+  - url: /engineering-education/android-recyclerviews/hero.jpg
+    alt: Android recyclerviews image
+---
+Loading vast amounts of data into normal views can lead to high CPU usage. This, in turn, may lead to the operating system forcibly shutting down the applications.
+In Android, it is common to display a list of data. With more data being presented and dynamically received, developers had to come up with compound views to help display the data. It has not been an easy task to say the least.
+<!--more-->
+#### Android RecyclerViews
+The Dynamic data (that we are referring to) can include data from a network or local database. This again comes with the added problem of resource management.
+
+That is where ListViews came into play. Developers were able to load text-based data to a compound view using an adapter to increase efficiency. However, this solution did not address all the problems encountered.
```

Comment:
This section was a bit confusing to me ( it maybe because I don't have the experience) but I had to get all the way to the bottom of the article to kind of understand what the 1st few paragraphs were trying to explain 🤔 
It may be helpful to revisit to make sure our thoughts are clearly explained to the audience. 🚀

Reply:
Let me run this section by some of the android developers I know and get the feedback back to you.